test/Microsoft.Extensions.Internal.Test/PropertyHelperTest.cs

```diff
@@ -718,6 +734,23 @@ private class Static
             public int Prop5 { get; set; }
         }
 
+#if NETCOREAPP2_2
+
+        private class RefStructProperties
+        {
+            public Span<bool> Span => throw new NotImplementedException();
+            public MyRefStruct UserDefined => throw new NotImplementedException();
+
+            public int Prop5 { get; set; }
+        }
+
+        private readonly ref struct MyRefStruct
+        {
+        }
+#elif NET461
+#else
+#error Unknown TFM - update the set of TFMs where we test for ref structs
```

Comment:
I've not seen this pattern before. Nice 👍

Reply:
yeah we tend to use this (at least in the MVC room) where we have if/defs so that whoevers updating at least has to think about it.